### PR TITLE
Update tip in 2024-04-02-shared-conda-tutorial.md

### PR DESCRIPTION
**Pull request by pvanheus**

### CODE REVIEW

**Comment by hexylena** [comment]
perhaps add yourself to the contributors? (and contributors.yaml)

File: _posts/2024-04-02-shared-conda-tutorial.md [hunk 1]
```diff
@@ -90,16 +90,16 @@ If you do not like having conda always active due to shell startup speed impact,
 By adding this function to your shell initialisation (`$HOME/.bashrc`), when you run a conda command like `conda activate x` it will first delete the function definition before sourcing the conda hook and running your desired conda command:
 
 ```bash
+CONDA_DIR=/home/user/arbeit/deps/miniconda3
```
Comment (pmenzel) [inline _posts/2024-04-02-shared-conda-tutorial.md:93]: I would replace that path with the path from the tutorial above.

**Comment by hexylena** [comment]
(build failure would be addressed by rebasing on main)

**Review by hexylena — APPROVED** [review]
Thank you for correcting the path

**Pull request merged**